.github/workflows/build_test.yml

```diff
@@ -436,7 +436,7 @@ jobs:
       run: |
         export CC=/usr/bin/gcc-12
         export CXX=/usr/bin/g++-12
-        python -m pip install -e .
+        python setup.py develop
```

Comment:
Shouldn't this use meson?

Reply:
This is a SIMD-related job, right? I left all of those on `setup.py` builds.

Reply:
I'll resolve this; there are a number of CI jobs still to move over, this is simply one of them. It's tracked in gh-23981.